src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Debug/LaunchProfile.cs

```diff
@@ -9,7 +9,7 @@ namespace Microsoft.VisualStudio.ProjectSystem.Debug
     /// <summary>
     /// Represents one launch profile read from the launchSettings file.
     /// </summary>
-    internal class LaunchProfile : ILaunchProfile, IPersistOption
+    internal class LaunchProfile : ILaunchProfile, ILaunchProfile2, IPersistOption
```

Comment:
Its interesting that when we added native & SQL debuggiong support we didn't add a new version of ILaunchProfile. I don't have a strong opinion on this, just calling out that its different to the last few properties.

Reply:
I'm moving these to the 'other' properties group, removing the new interfaces.